### DISCUSSION TITLE
Announcing Roscal: Open Security Controls Assessment Language Toolbox

### DISCUSSION THREAD

**Discussion by gborough** [usnistgov/OSCAL#1986] [Show and tell]
Hi everyone, I would like to contribute to this great project by releasing roscal, the Open Security Controls Assessment Language Toolbox. The project aims to develop a set of tooling for the OSCAL ecosystem and focuses on automation and integration with a variety of environments, with goals to enable greater data gathering capabilities for OSCAL model file building, conversion and normalisation between existing cyber security standards and schemas, and hopefully one day able to achieve automatic security and control enforcement to certain degree based on these model files. Initial release contains a CLI tool for models manipulation and a Rust library for programmatically building models, more work to commence soon.

I will keep it brief here so looking forward to more discussion. You can locate the project here:

Source code: https://github.com/gborough/roscal

Documentation: https://crates.io/crates/roscal_lib

**Comment by iMichaela** [comment]
@gborough - This is an awesome initiative!! I think you should present this work to our community to bring awareness and find collaborators.  
Also - don't forget to list the project on [awesome-oscal](https://github.com/oscal-club/awesome-oscal) and [OSCAL.io/tools](https://oscal.io/tools/).
